_maps/map_catalogue.txt

```diff
@@ -499,6 +499,10 @@ Find the key for using this catalogue in "map_catalogue_key.txt"
 		Size = (x = 37)(y = 43)(z = 1)
 		Tags = "Medium Combat Challenge", "Medium Loot", "Liveable"
 
+		File Name = "_maps\RandomRuins\BeachRuins\beach_xenolab.dmm"
+		Size = (x = 80)(y = 70)(z = 1)
```

Comment:
X should be no greater than 75 otherwise we get _extra_ ruin smushing from ships

Reply:
Done.